pyanaconda/ui/lib/subscription.py

```diff
@@ -108,6 +111,40 @@ def check_cdn_is_installation_source(payload):
         # DNF payload at the moment
         return False
 
+
+def check_simple_content_access_is_enabled():
+    """Check if the currently used Red Hat account has Simple Content Access enabled.
+
+    :type subscription_proxy: proxy for the Subscription DBus module
+    """
+    subscription_proxy = SUBSCRIPTION.get_proxy()
+
+    task_path = subscription_proxy.ParseOrganizationDataWithTask()
+    task_proxy = SUBSCRIPTION.get_proxy(task_path)
+    task.sync_run_task(task_proxy)
+    # GetResult() returns a variant, which we need to unwrap and then
+    # convert the list of structs to a list of actual data objects we can easily
+    # work with.
+    org_data_structs = unwrap_variant(task_proxy.GetResult())
+    org_data_list = OrganizationData.from_structure_list(org_data_structs)
+    if org_data_list:
+        if len(org_data_list) > 1:
+            log.warning("subscription thread: more than one organization returned "
+                        "for Red Hat account currently in use")
+            log.warning(org_data_list)
+        # On hosted Candlepin there should be exactly one organization per user account,
+        # on custom Candlepin instances there could be (and often are).
+        # For now just always pick the first one as the authoritative organization.
+        return org_data_list[0].simple_content_access_enabled
```

Comment:
That's a lot of code to simply check that there is an organization with the simple content access enabled. Are there some specific plans for using `OrganizationData` outside of this use case?